interpreter/binary/encode.ml

```diff
@@ -875,36 +875,66 @@ struct
     section 12 len (List.length datas) Free.((module_ m).datas <> Set.empty)
 
   (* Custom section *)
-  let custom (n, bs) =
+  let custom c =
+    let Custom.{name = n; content; _} = c.it in
     name n;
-    put_string s bs
+    put_string s content
 
-  let custom_section n bs =
-    section 0 custom (n, bs) true
+  let custom_section place c =
+    let here = Custom.(compare_place c.it.place place) <= +1 in
```

Comment:
Likely my own code, but this condition seems dubious. The result of a compare is one of -1, 0, or +1, so the condition would always be true. I believe this should be `<= 0`. Can you change and see if that works?

Reply:
yes, it was broken. Maybe a mistake during a rebase, since I think that it used to work during testing. Good catch.
Fixed.